app/js/handlebarsQuestionTable.js

```diff
@@ -0,0 +1,102 @@
+let baseUrl = getBaseUrl()
```

Comment:
Try to minimise unecessary 'floaters' just call getBaseUrl() wherever you need it

Reply:
Fixed